lib/create-http-client.js

```diff
@@ -20,6 +20,7 @@ const HOST_REGEX = /^(?!\w+:\/\/)([^\s:]+\.[^\s:]+)(?::(\d+))?(?!:)$/
  * @prop {Object=} httpsAgent - HTTPS agent for node
  * @prop {Object=} proxy - Axios proxy config
  * @prop {Object=} headers - Additional headers
+ * @prop {function=} logHandler - A log handler function to process given log messages & errors. Receives the log level (error, warning & info) and the actual log data (Error object or string). (default: Check source)
```

Comment:
Small nit:
> default: check the source of this function

Because it will be rendered into GH pages, where you don't have the source immediately available. So "source" would refer to the whole repo.

Reply:
Good catch, so I fall back to the actual URL to the file which should not change :)